src/main/resources/static/css/styles.css

```diff
@@ -237,22 +287,158 @@ img.user-profile-avatar {
     padding-left: 10px;
 }
 
-/*ERROR PAGE STYLING*/
-.error-container, .error-404-container, .error-4xx-container {
-    height: 50vh;
+/*Styling nav link anchor tags*/
+a.nav-link {
+    color: var(--grey-color);
+}
+
+a.nav-link:hover {
+    color: var(--blush-color);
+}
+
+/*Dropdown Menu Stying*/
+.dropdown-menu {
+    border: none;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+}
+
+/*Removing box around hamburger icon when it is clicked*/
+.navbar-toggler:focus,
+.navbar-toggler:active,
+.navbar-toggler-icon:focus {
+    outline: none;
+    box-shadow: none;
+}
+
+/* Navbar Hamburger Styling */
+.hamburger-animated-icon {
+    width: 30px;
+    height: 20px;
+    position: relative;
+    margin: 0px;
+    -webkit-transform: rotate(0deg);
+    -moz-transform: rotate(0deg);
+    -o-transform: rotate(0deg);
+    transform: rotate(0deg);
+    -webkit-transition: .5s ease-in-out;
+    -moz-transition: .5s ease-in-out;
+    -o-transition: .5s ease-in-out;
+    transition: .5s ease-in-out;
+    cursor: pointer;
+}
+
+.hamburger-animated-icon span {
+    display: block;
+    position: absolute;
+    height: 3px;
     width: 100%;
+    border-radius: 9px;
+    opacity: 1;
+    left: 0;
+    -webkit-transform: rotate(0deg);
+    -moz-transform: rotate(0deg);
+    -o-transform: rotate(0deg);
+    transform: rotate(0deg);
+    -webkit-transition: .25s ease-in-out;
+    -moz-transition: .25s ease-in-out;
+    -o-transition: .25s ease-in-out;
+    transition: .25s ease-in-out;
```

Comment:
So excited to see all of these style updates on the site after I pull down :D

Reply:
Same! Our site is going to look great! 🤗